test-requirements.txt

```diff
@@ -0,0 +1,4 @@
+nose
+mock<=1.0.1
```

Comment:
They dropped Python 2.6 compatibility.
https://github.com/testing-cabal/mock/commit/8a2e8709502c30e2dd0838125bc5c34fec35f451
However, I see that they later returned it, so this is probably not necessary anymore. We would have to somehow exclude 1.1.0..1.1.4.
https://github.com/testing-cabal/mock/commit/a6367a9a2b6166d7d032ec91288294ec47177649